lib/maintenance_tasks.rb

```diff
@@ -52,6 +52,16 @@ module MaintenanceTasks
   #     app's config/storage.yml.
   mattr_accessor :active_storage_service
 
+  # @!attribute backtrace_cleaner
+  #   @scope class
+  #
+  #   The Active Support backtrace cleaner that will be used to clean the
+  #   backtrace of a Task that errors.
+  #
+  #   @return [ActiveSupport::BacktraceCleaner, nil] the backtrace cleaner to
+  #     use when cleaning a Run's backtrace.
+  mattr_accessor :backtrace_cleaner
```

Comment:
Tried to use `default: Rails.backtrace_cleaner` here to avoid having to check for it in `#persist_error`, but unfortunately `Rails.root` is not set at this point, so when the `Rails.backtrace_cleaner` gets set (and memoized) here, it's `@root` is wrong (nil instead of the dummy app) and this throws off the traces.